docs/release-notes/NuGet-5.0-RTM.md

```diff
@@ -0,0 +1,156 @@
+
+* PMC: Update-Package reinstall fails ("Unable to find package") on delisted packages. - [#7268](https://github.com/NuGet/Home/issues/7268)
+
+* Add third party notice in our repo and VSIX - [#7409](https://github.com/NuGet/Home/issues/7409)
+
+* NuGet.VisualStudio.IVsPackageInstaller.InstallPackage should install latest version when no version given - [#7493](https://github.com/NuGet/Home/issues/7493)
+
+* --interactive support for dotnet nuget push - [#7519](https://github.com/NuGet/Home/issues/7519)
+
+* When restoring with lock file, NU1603 warning shouldn't be raised. - [#7529](https://github.com/NuGet/Home/issues/7529)
+
+* NuGet should not print project path during restore with minimal logging - [#7647](https://github.com/NuGet/Home/issues/7647)
+
+* --interactive support for dotnet remove package - [#7727](https://github.com/NuGet/Home/issues/7727)
+
+* Add back NuGet.Packaging.Core with TypeForwardedTo attrs - [#7768](https://github.com/NuGet/Home/issues/7768)
+
+* plugins_cache needs shorter path to work well - [#7770](https://github.com/NuGet/Home/issues/7770)
+
+* Prefer path for msbuild discovery if user didn't ask for specific msbuild version - [#7786](https://github.com/NuGet/Home/issues/7786)
+
+* nuget.exe /? should list correct msbuild versions - [#7794](https://github.com/NuGet/Home/issues/7794)
+
+* NuGet.targets(498,5): error : Could not find a part of the path '/tmp/NuGetScratch - on mono - [#7793](https://github.com/NuGet/Home/issues/7793)
+
+* restore unnecessarily enumerates the contents of all versions of referenced package in the machine cache - [#7639](https://github.com/NuGet/Home/issues/7639)
+
+* MSBuild auto-detection always selects 16.0 after installing VS 2019 Preview - [#7621](https://github.com/NuGet/Home/issues/7621)
+
+* dotnet list package on a solution outputs duplicate entries for framework - [#7607](https://github.com/NuGet/Home/issues/7607)
+
+* Exception "Empty path name is not legal" when calling IVsPackageInstaller.InstallPackage on old projects and packages folder does not exist. - [#5936](https://github.com/NuGet/Home/issues/5936)
+
+* msbuild /t:restore minimal verbosity should be more minimal - [#4695](https://github.com/NuGet/Home/issues/4695)
+
+* VS 16.0's NuGet UI has unreadable tabs due to color problems - [#7735](https://github.com/NuGet/Home/issues/7735)
+
+* NuGet.Core & NuGet.Clients License.txt clarification - [#7629](https://github.com/NuGet/Home/issues/7629)
+
+* Restore unnecessarily enumerates global package folder in attempt to determine type - [#7596](https://github.com/NuGet/Home/issues/7596)
+
+* Errors from lock file enforcement should show up in Error List Window - [#7429](https://github.com/NuGet/Home/issues/7429)
+
+* Fix NuGet.Configuration issues - [#7326](https://github.com/NuGet/Home/issues/7326)
+
+* Adapt to MSBuild updating it's install location.  - [#7325](https://github.com/NuGet/Home/issues/7325)
```

Comment:
```suggestion
* Adapt to MSBuild updating its install location.  - [#7325](https://github.com/NuGet/Home/issues/7325)
```

Reply:
i've edited the title of issue

@karann-msft  - please manually fix